docs/requirements.txt

```diff
@@ -1,13 +1,15 @@
 # JupyterLite (kernel requires specific pyodide version)
-jupyterlite-pyodide-kernel==0.1.0
-jupyterlite[piplite]==0.1.0
-jupyterlite-sphinx==0.8.0
+# This kernel requires a specific pyodide version (see pyodide-build)
+jupyterlite-pyodide-kernel==0.2.*
+jupyterlite[piplite]==0.2.*
+jupyterlite-sphinx==0.11.*
 
-sphinx>=4.5.0,<5.0.0
+sphinx==7.*
```

Comment:
Pin to a version? The [admonitions](https://iscinumpy.dev/post/bound-version-constraints/) against `<` must apply to `=` as well[^1]; is there a reason to think that Sphinx 8 would be bad?

[^1]: If they don't, then I don't understand the issue.

Reply:
> Right now we'll pin it, but we should upgrade to sphinx 7.

Okay!

Reply:
For application-like uses, we can be very restrictive e.g. pin exactly. I'm not in favour of going _that_ far; we just want our docs to break less often.